build.gradle

```diff
@@ -70,7 +70,8 @@ dependencies {
     nativeZip wpi.deps.vendor.jni(wpi.platforms.roborio)
     nativeDesktopZip wpi.deps.vendor.jni(wpi.platforms.desktop)
 
-    testImplementation 'junit:junit:4.12'
+    implementation 'junit:junit:4.12'
```

Comment:
This should also be `testImplementation`

Reply:
Thought I changed it back after I removed the class I wrote to run tests in a single instance. Will change that.